activation/nipost.go

```diff
@@ -197,6 +197,7 @@ func (nb *NIPostBuilder) BuildNIPost(ctx context.Context, challenge *types.NIPos
 		log.Time("poet proof deadline", poetProofDeadline),
 		log.Stringer("publish epoch", pubEpoch),
 		log.Stringer("target epoch", challenge.TargetEpoch()),
+		log.Any("config", nb.poetCfg),
```

Comment:
This will print redundant information here (mostly what's already logged at the other fields). I think it would be better to instead log the relevant config parameters in the `poetClient`, like MaxRetries and RetryDelay.